core/domain/exp_domain.py

```diff
@@ -6575,7 +6545,7 @@ def __init__(
         self.param_specs = param_specs
         self.param_changes = param_changes
         self.auto_tts_enabled = auto_tts_enabled
-        self.correctness_feedback_enabled = correctness_feedback_enabled
+        self.correctness_feedback_enabled = feconf.DEFAULT_CORRECTNESS_FEEDBACK_ENABLED # pylint: disable=line-too-long
```

Comment:
You should be dropping the entire field and property from the domain layer (and pretty much all layers). Ditto throughout this PR.

We just need to make sure that items that include correctness_feedback_enabled still parse correctly. These items might be in the datastore (it should be fine if you never read the field when fetching from the datastore and converting it into a domain object), the memcache (we should again ignore that field when we convert to a domain object), and localStorage (I don't think we store this info there though).

Reply:
Done